### DISCUSSION TITLE
Add new libraries

### DISCUSSION THREAD

**Discussion by rcmannepalli** [modular/modular#158] [General]
How can I install python libraries that doesn't come default with Mojo? and how can we create new libraries using Mojo?

**Comment by mfranzon** [comment]
In these notebooks, from my understanding, there is not the possibility to install packages directly on the kernel. And I think the same for creating new libraries, as discussed [here](https://github.com/modularml/mojo/discussions/166), seems that there is not an easy way to extend the modules/libraries.

**Comment by rcmannepalli** [comment]
Thank you. I believe these features are coming soon. It is basic structure of any programming language. Looking forward to these features available.

**Reply by mfranzon** [reply]
Indeed, totally agree!

**Comment by iamtimdavis** [comment]
Please also see our survey of packages you want to install -> https://github.com/modularml/mojo/discussions/173

We are working hard on local execution. Please see -> https://docs.modular.com/mojo/faq.html#why-cant-i-run-mojo-locally
